ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -192,31 +193,58 @@ func getCommPf(mode string, linkList []netlink.Link) (netlink.Link, error) {
 	return pf, nil
 }
 
+// It can take time for network-manager's state for each interface, to become
+// activated, when IP address is set, which can cause the IP address to not stick
+// Removing the host-acc comm channel interface out of network-manager's mgmt.
+func (e *ExecutableHandlerImpl) nmcliSetup(link netlink.Link) error {
+	intfName := link.Attrs().Name
+	output, err := utils.ExecuteScript(`nmcli device set ` + intfName + ` managed no`)
+	if err != nil {
+		log.Errorf("nmcli err->%v, output->%v\n", err, output)
+		return fmt.Errorf("nmcli err->%v, output->%v\n", err, output)
```

Comment:
Is there a possibility that this command can return an error let's say if the interface is already unmanaged, or some other scenario where we want to ignore the error? I have a concern where the error will be propagated and other functionality after the `configureChannel` returns an error, won't be executed

Reply:
I tested this, if it is already unmanaged, then it still returns success. For RHEL, networkmanager daemon runs by default, and use of nmcli will work fine. But if the OS is different like ubuntu, which uses systemd-networkd, which uses a different client, then nmcli is likely to error out here.  Hi Vadivel, @vkannapp , I suppose we would like to know...if it errors out here or can we ignore any error for nmcli. Please chime in.

Reply:
@saynb Discussed with Vadivel, and added nmcli general status, which checks if Network Manager is running, before running the nmcli command to remove the interface from being managed. If it looks fine, you can approve this.